programmers/비밀지도.js

```diff
@@ -0,0 +1,25 @@
+function solution(n, arr1, arr2) {
+    
+    const output = [];
+    let result = [];
+    
+    for (let i = 0; i < n; i++) {
+        arr1[i] = arr1[i].toString(2);
+        arr2[i] = arr2[i].toString(2);
+        if (arr1[i].length < n) {
+            while (arr1[i].length < n) arr1[i] = 0 + arr1[i];
+        }
+        if (arr2[i].length < n) {
+            while (arr2[i].length < n) arr2[i] = 0 + arr2[i];
+        }
+        for (let j = 0; j < n; j++) {
+            if (arr1[i][j] === '0' && arr2[i][j] === '0') result.push(' ');
+            if (arr1[i][j] === '1' || arr2[i][j] === '1') result.push('#')
+        }
```

Comment:
```suggestion
         for (let j = 0; j < n; j++) {
            (arr1[i][j] === '0' && arr2[i][j] === '0') ? result.push(' ') : result.push('#');
        }
```
이렇게 삼항 연산자로 하는게 더 간단하네요